ee/agent/internal/informer_modules.go

```diff
@@ -0,0 +1,86 @@
+package internal
+
+import (
+	"reflect"
+
+	"github.com/formancehq/stack/components/agent/internal/generated"
+	"github.com/formancehq/stack/libs/go-libs/logging"
+	"google.golang.org/protobuf/types/known/structpb"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/tools/cache"
+)
+
+func getStatus(logger logging.Logger, unstructuredModule *unstructured.Unstructured) *structpb.Struct {
+	status, found, err := unstructured.NestedMap(unstructuredModule.Object, "status")
+	if !found || err != nil {
+		return &structpb.Struct{Fields: map[string]*structpb.Value{}}
+	}
+
+	protoStatus, err := structpb.NewStruct(status)
+	if err != nil {
+		logger.Errorf("Unable to convert status to struct: %s", err)
+		return &structpb.Struct{Fields: map[string]*structpb.Value{}}
```

Comment:
Without returning an error, the agent will send an empty status to membership.
I think the agent should not send anything in this case, to avoid bad status on membership.